Public/Src/App/UnitTests/Bxl/ServerDeploymentTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using BuildXL;
+using BuildXL.Engine;
+using BuildXL.Utilities;
+using Test.BuildXL.TestUtilities.Xunit;
+using Xunit;
+
+namespace Test.Bxl
+{
+    public class ServerDeploymentTests : TemporaryStorageTestBase
+    {
+        [Fact]
+        public void TestMissingManifestDirectory()
+        {
+            string deploymentDir;
+            XAssert.IsTrue(ServerDeployment.IsServerDeploymentOutOfSync(TemporaryDirectory, null, out deploymentDir));

```

Comment:
As a shortcut, you can do 
```suggestion
            XAssert.IsTrue(ServerDeployment.IsServerDeploymentOutOfSync(TemporaryDirectory, null, out var deploymentDir));
```
and then delete `string deploymentDir;` above.